cli.py

```diff
@@ -1,7 +1,7 @@
 import click
 
 import ml
-from ml.base import MLConfig
+from ml.base import GlobalConfig
 from ml.base.estimator import MLEstimator
 from ml.squeezeseg.cli import npy2tfrecord
```

Comment:
this import is very ugly @FXuZ do you have a better solution to make sure the class definition for specific io / trainers etc run before cli's logic is called?

Reply:
procrastinated to issue #27